site/content/contribute/getting-started/code-review.md

```diff
@@ -38,47 +38,65 @@ If you are a core committer seeking a review
+        - If your change primarily touches another team's codebase, consider their product mananger.
+        - If in doubt, assign any product mananger and comment about the uncertainty. The product mananger may reassign as appropriate.
+    * [Wait](#if-you-are-awaiting-a-review) for their review to complete before continuing so as to avoid churn if changes are requested.
+    * Remove the `1: PM Review` label only when their review is done.
+        - Product managers ensure the changes meet [user experience guidelines](https://docs.mattermost.com/developer/fx-guidelines.html).
+    * If your changes do not affect the user experience, you may remove `1: PM Review` immediately.
+6. After PM review, request a review from two [core committers](/contribute/getting-started/core-committers/).
+    * The choice of core committers is up to you.
+        - When picking your first core committer, consider someone with domain expertise relative to your changes. Sometimes GitHub will recommend a recent editor of the code, but often you must rely on your own intuition from past interactions.
+        - When picking your second core committer, consider someone from another team who may have expertise in the language you're using or the problem you're solving, even if they aren't intricately familiar with the codebase. This can provide a fresh set of eyes on the code to reveal blindspots that are not biased by hitting deadlines, and helps expose the team to new parts of the code to help spread out domain knowledge. This may not make sense for every pull request but is a practice to keep in mind.
+        - Don't be afraid to pick someone who gives "hard" reviews. Code review feedback is never a personal attack: it should "sharpen" the skills of both the author and the reviewers, not to mention improving the quality of the product.
+        - Try to avoid assigning the same person to all of your reviews unless they are related.
+        - When in doubt, ask for recommendations on our community server.
+    * [Wait](#if-you-are-awaiting-a-review) for their review to complete before continuing so as to avoid churn if changes are requested.
+    * Remove the `2: Dev Review` label only when these reviews are done.
+        - The last developer to review may choose to help here and speed up the process.
+7. After Dev review, assign a [QA tester](/contribute/getting-started/core-committers/#qa-testers).
+    * The choice of QA tester is up to you.
+        - In most cases, choose the QA tester embedded with your team.
+        - If your change primarily touches another team's codebase, consider their QA tester.
+        - If in doubt, assign any QA tester and comment about the uncertainty. The QA tester may reassign as appropriate.
+    * [Wait](#if-you-are-awaiting-a-review) for their review to complete before continuing.
+        - It is the QA tester's responsibility to determine the scope of required testing, if any.
```

Comment:
Would it make sense to include a note that in some cases QA review might happen in parallel with PM and/or Dev reviews? Considered suggesting it earlier in the doc, like in the PM section, but given that it is an exception, and a QA-focused exception, maybe it would fit in this section. 

Something like "In some cases it may be agreed during planning that QA should review in parallel with PM or Dev reviews." ?

Reply:
I think it makes sense to stick to the happy path for documentation purposes -- already so much text here.